src/main/java/nextstep/subway/domain/Favorite.java

```diff
@@ -0,0 +1,55 @@
+package nextstep.subway.domain;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+
+@Entity
+public class Favorite {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private Long memberId;
+
+    @ManyToOne(cascade = CascadeType.PERSIST)
+    @JoinColumn(name = "source_id")
+    private Station source;
+
+    @ManyToOne(cascade = CascadeType.PERSIST)
+    @JoinColumn(name = "target_id")
+    private Station target;
+
+    protected Favorite() {
+    }
+
+    public Favorite(Long memberId, Station source, Station target) {
+        this.memberId = memberId;
+        this.source = source;
+        this.target = target;
+    }
+
+    public boolean isSameMember(Long memberId) {
```

Comment:
member를 직접 받는게 어떨까요?

```suggestion
    public boolean isSameMember(Member member) {
```